rmw_fastrtps_shared_cpp/include/rmw_fastrtps_shared_cpp/qos.hpp

```diff
@@ -28,6 +28,10 @@ class PublisherAttributes;
 }  // namespace fastrtps
 }  // namespace eprosima
 
+RMW_FASTRTPS_SHARED_CPP_PUBLIC
+bool
+is_valid_qos(const rmw_qos_profile_t & qos_policies);
```

Comment:
any reason for moving this declaration?

Reply:
I felt it would be more consistent with the rest of the `rmw` implementations that the `dds_qos_to_rmw_qos()` follows `get_datareader_qos()`. `is_valid_qos()` is also the first function among the ones declared here to be called in a normal program flow.

Reply:
Sounds good